RELEASE/scripts/autoscend/paths/actually_ed_the_undying.ash

```diff
@@ -1443,7 +1443,7 @@ boolean LM_edTheUndying()
 		return true;
 	}
 	// L8 quest is all 1 Ka zones for Ed (unlikely to survive Ninja Snowmen Assassins so they don't count)
-	if (L8_trapperStart() || L8_trapperGround() || L8_trapperGroar())
+	if (L8_trapperQuest())
 	{
 		return true;
 	}
```

Comment:
* L8_trapperStart() is no longer necessary. starting as part of L8_trapperGround()
* ed was relying on backtracking code in L8_trapperGroar() which went and executed L8_trapperExtreme() if you managed to reach that function without doing the ninjas already. since this backtracking code has been removed I switched to L8_trapperQuest() and added a check on ninja's to have HC ed skip them (softcore ed wants to pull the ninja gear)